src/core/task/task.cc

```diff
@@ -57,6 +58,18 @@ void LegateTaskRegistrar::record_variant(TaskID tid,
 
 void LegateTaskRegistrar::register_all_tasks(Runtime* runtime, LibraryContext& context)
 {
+  for (auto& taskIdx : Core::opIDs){
+    auto newID = context.get_task_id(taskIdx.first);
+    Core::cpuDescriptors.insert(std::pair<int64_t, LegateVariantImpl>((int64_t) newID, taskIdx.second));
+  }
+
+  for (auto& taskIdx : Core::gpuOpIDs){
+    auto newID = context.get_task_id(taskIdx.first);
+    Core::gpuDescriptors.insert(std::pair<int64_t, LegateVariantImpl>((int64_t) newID, taskIdx.second));
+  }
+
+
```

Comment:
register each legate tasks function pointer, so they can be accessed from a fused task